src/quo2/components/settings/settings_item/view.cljs

```diff
@@ -91,7 +91,7 @@
   [{:keys [action action-props blur? theme]}]
   [rn/view {:style {:margin-left 12}}
    (case action
-     :arrow    [icon/icon :i/chevron-right (style/color blur? theme)]
+     :arrow    [icon/icon (or (:icon action-props) :i/chevron-right) (style/color blur? theme)]
```

Comment:
removing this default might break other uses? perhaps you have to set it as a default value here?

Reply:
I am not removing anything, I am adding a default :)